src/Microsoft.AspNetCore.Blazor.Templates/content/BlazorServerSide-CSharp/BlazorServerSide-CSharp.Client/_ViewImports.cshtml

```diff
@@ -0,0 +1,5 @@
+﻿@using System.Net.Http
+@using Microsoft.AspNetCore.Blazor.Layouts
+@using Microsoft.AspNetCore.Blazor.Routing
+@using BlazorServerSide_CSharp.Client
+@using BlazorServerSide_CSharp.Client.Shared
```

Comment:
Is there anything in the `.Shared` project? Should we still have it in the server-side sample?

Reply:
I'm in favor of removing the shared project in this case

Reply:
Oh yes, well remembered!

Reply:
It's not a shared project, its the `/shared/` folder in the client project.